Patches/Starship Troopers Arachnids/Resources.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>[1.3] Starship Troopers Arachnids Ver6.3</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+				<!-- Arachnid Leg -->
+				<li Class="PatchOperationAdd">
+					<xpath>/Defs/ThingDef[defName="ArachnidLeg"]/statBases</xpath>
+					<value>
+						<Bulk>20.53</Bulk>

```

Comment:
This is extremely high bulk for something that's supposed to be a melee weapon. For reference, a longsword has a bulk of 8.